powerhome/playbook
# [PCA-105] Add size prop to the button kit/fixing bug on previous PR

#### Screens

<img width="1758" alt="Playbook_Design_System_and_jelenadurovic_Jelenas-MacBook-Pro___Desktop_Nitro_playbook_playbook_playbook" src="https://user-images.githubusercontent.com/82796889/151225551-38bd0769-747d-49d2-9682-d3eb2b795bfb.png">


#### Breaking Changes

No. This PR is a fixing a bug where if you've added `sm` to the classname button size will shrink and button size will be affected even if you just wanted to add `mb_sm`.

#### Runway Ticket URL

[[INSERT URL]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-105)

#### How to test this

Add any margin with `sm` value and make sure that the size of the button doesn't change. 

#### Checklist:
